PULL_REQUEST_TEMPLATE

```diff
@@ -19,4 +19,5 @@ Don't forget to add a prefix to the pr title in this format
 - [ ] I wrote Unit Tests that confirm the expected behaviour
 - [ ] I updated only the [english localization source files](Client/Ecosia/L10N/es.lproj) if needed
 - [ ] I added the `// Ecosia:` helper comments where needed
+- [ ] I made sure that any change to the Analytics events included in PR won't alter current analytics (e.g. new users, upgrading users)
```

Comment:
I would like to amend something around those lines: `In the implementation of the analytics event I considered testing both scenarios, new install and update`

Reply:
That's a good one 👍 , but we may have other analytics that would still make unexpected behavior for scenarios not linked with new install / update specifically. That's the reasons of leaving it quite open, appointing at the same time to the Team involved in any clarification if needed.